lib/plugin.js

```diff
@@ -179,6 +180,68 @@ const setupProgress = (axios) => {
   axios.defaults.onDownloadProgress = onProgress
 }<% } %>
 
+<% if (options.proxyCookies) { %>
+const proxyCookies = (ctx, axios, response) => {
+  const parseSetCookies = cookies => {
+    return cookies
+      .map(cookie => cookie.split(';')[0])
+      .reduce((obj, cookie) => ({
+        ...obj,
+        ...parseCookies(cookie)
+      }), {})
+  }
+
+  const serializeCookies = cookies => {
+    return Object
+      .entries(cookies)
+      .map(([name, value]) => `${name}=${encodeURIComponent(value)}`)
+      .join('; ')
+  }
+
+  const mergeSetCookies = (oldCookies, newCookies) => {
+    const cookies = new Map()
+
+    const add = setCookie => {
+      const cookie = setCookie.split(';')[0]
+      const name = Object.keys(parseCookies(cookie))[0]
+
+      cookies.set(name, cookie)
```

Comment:
It should be `cookies.set(name, setCookie)`. 
Otherwise, cookie might be duplicated by HttpOnly/Domain/SameSite/Max-Age/etc parameters.

Reply:
For example:
1st row - cookie from backend
2nd row - cookie from ssr-cookie-proxy

![image](https://user-images.githubusercontent.com/9816112/202607294-a3d79d38-b526-4712-8e62-1344f497c927.png)